electrical/electrical_build.md

```diff
@@ -0,0 +1,252 @@
+
+* [Quadrature Encoder Basics](https://www.dynapar.com/Technology/Encoder_Basics/Quadrature_Encoder/)
+
+In this project, we use absolute encoders on the corner wheels to get absolute measurements on where the corner wheels are turned/pointed. We use quadrature encoders at each of the driving wheels to get speed measurements.
+# 2. Powering Components
+This section will go through the process of powering each of the components from the battery. During this project you will need to cut, strip, and solder wires to extend and split electrical connections. It is important to understand how to do this safely so that you are protecting your system from electrical shorts. Below are a few quick links on these skills, however it is crucial that you fully understand these processes before proceeding.
+
+* [SparkFun Stripping Wire](https://learn.sparkfun.com/tutorials/working-with-wire/how-to-strip-a-wire) 
+* [Splicing Wire](https://www.wikihow.com/Splice-Wire)
+* [YouTube Making Connections and Heat Shrink Tubing](https://www.youtube.com/watch?v=Y8wjv6lj5KU)
+
+**Note: Please read through this entire section and understand it fully before starting to connect devices and electronics together.**
+
+## 2.1. Switch, Volt Meter, and Control Board
+There are a couple of very important things to be aware of as you are making these connections, as with any project where you are working with batteries or electrical current:
+**THE BATTERY IS LIVE AT ALL TIMES.** This means that at any point, if the two terminals of the battery come in contact they will cause an electrical short which could be very dangerous. This may trigger the protection circuit in the battery that we chose and prevent extensive damage, but we do not want to rely on that for safety. Make sure to keep the battery terminals of the +V and GND separated at all times with no possibility of contact.
+
+The battery connectors will only fit together one way. That means you have to make sure your connections are correct the first time when soldering them in: make sure to verify this before connecting anything to the battery.
+The switch we use is a Single Pole - Single Throw switch. This means that either the two terminals are connected together in one switch position or there is a physical disconnect when the switch is in the other position. Make sure to test the switch and know which are its **ON** and **OFF** positions previous to connecting it to anything. An easy way to test the switch is to use the ’Diode’ setting on your Digital Multimeter (DMM).
+It is extremely important to plug the battery in the correct direction into the volt meter as well, as plugging it in backwards (even for an instant) could damage or destroy the volt meter.
```

Comment:
To call this out better, let's do this. See rendered version at: https://github.com/apollokit/open-source-rover/blob/warning_fix/electrical/electrical_build.md#21-switch-volt-meter-and-control-board

```suggestion

| :exclamation: :exclamation: :exclamation: Battery Safety :exclamation: :exclamation: :exclamation: |
|:---------------------------|
| There are a couple of very important things to be aware of as you are making these connections, as with any project where you are working with batteries or electrical current: |
| **THE BATTERY IS LIVE AT ALL TIMES.** This means that at any point, if the two terminals of the battery come in contact they will cause an electrical short which could be very dangerous. This may trigger the protection circuit in the battery that we chose and prevent extensive damage, but we do not want to rely on that for safety. Make sure to keep the battery terminals of the +V and GND separated at all times with no possibility of contact. |
| The battery connectors will only fit together one way. That means you have to make sure your connections are correct the first time when soldering them in: make sure to verify this before connecting anything to the battery. |
| The switch we use is a Single Pole - Single Throw switch. This means that either the two terminals are connected together in one switch position or there is a physical disconnect when the switch is in the other position. Make sure to test the switch and know which are its **ON** and **OFF** positions previous to connecting it to anything. An easy way to test the switch is to use the ’Diode’ setting on your Digital Multimeter (DMM). |
| It is extremely important to plug the battery in the correct direction into the volt meter as well, as plugging it in backwards (even for an instant) could damage or destroy the volt meter.  |
| :exclamation: :exclamation: :exclamation: Battery Safety :exclamation: :exclamation: :exclamation: |

```